src/v2/Apps/Partner/partnerRoutes.tsx

```diff
@@ -87,6 +88,36 @@ export const partnerRoutes: RouteConfig[] = [
         prepare: () => {
           ContactRoute.preload()
         },
+        query: graphql`
+          query partnerRoutes_ContactQuery($partnerId: String!) {
+            partner(id: $partnerId) @principalField {
+              ...Contact_partner
+              locations: locationsConnection(first: 50) {
+                totalCount
+              }
```

Comment:
Not sure that it's correct. I added `locationsConnection` directly to the query because I couldn't get `partner.locations.totalCount` value below in the render function. Without `locationsConnection` locations field in the partner field was undefined.

@sweir27 @dzucconi Do you have a better variant for that?

Reply:
Nope! This makes sense to me.